packages/ember/tests/routing/basic_test.js

```diff
@@ -321,90 +321,6 @@ QUnit.test('Model passed via renderTemplate model is set as controller\'s model'
   equal(jQuery('p:contains(emberjs)', '#qunit-fixture').length, 1, 'Passed model was set as controllers model');
 });
 
-test('Renders correct view with slash notation', function() {
```

Comment:
[Here](https://github.com/asakusuma/ember.js/blob/45d0a7957e1b22233ab3f94bba8fc652306c7bd3/packages/ember/tests/routing/basic_test.js#L1876) is the analogous template test